developer-cli/Utilities/ProcessHelper.cs

```diff
@@ -22,7 +22,7 @@ public static void OpenBrowser(string url)
     {
         if (Configuration.IsWindows)
         {
-            StartProcess($"start {url}");
+            StartProcessWithSystemShell(url);
```

Comment:
Can this be simplified by just calling `StartProcess()` directly with the URL as the `FileName`, and then reverting the changes to `CreateProcessStartInfo()`?

```suggestion
           StartProcess(new ProcessStartInfo{ FileName = url });
```

Reply:
Yes, although it needs to be: 
```suggestion
            StartProcess(new ProcessStartInfo{ FileName = url, UseShellExecute = true });
```
I have pushed the simplified version.

Reply:
Yes, although it needs to be:
```C#
StartProcess(new ProcessStartInfo { FileName = url, UseShellExecute = true });
```
I have pushed the simplified version.